src/app/shared/enum/role.ts

```diff
@@ -2,24 +2,21 @@ export enum Role {
   all = 'all',
   unauthorized = 'unauthorized',
   provider = 'provider',
+  // TODO: Remove providerDeputy when position will be implemented, adjust checks
+  providerDeputy = 'providerdeputy',
+  employee = 'employee',
   parent = 'parent',
   techAdmin = 'techadmin',
   ministryAdmin = 'ministryadmin',
   regionAdmin = 'regionadmin',
   areaAdmin = 'areaadmin'
 }
 
-export enum Subrole {
-  None = 'None',
-  ProviderDeputy = 'ProviderDeputy',
-  ProviderAdmin = 'ProviderAdmin'
-}
-
 export enum EntityType {
   provider = 'provider',
   workshop = 'workshop',
-  ProviderAdmin = 'provideradmin',
-  ProviderDeputy = 'provider',
+  providerDeputy = 'provider',
```

Comment:
I would like to avoid using the same values for enums.
This was before your changes, but could you please check if it's not difficult to remove this

Reply:
Possibly, in the past this enum was dedicated to be used in filtering ratings by type. However, in our project it's always `workshop` type is used and for provider `rating` is always returned within the provider info.
Backend also uses only `workshop` for rating https://github.com/ita-social-projects/OoS-Backend/blob/87d225785d61b617b627b958179417313ab63ef6/OutOfSchool/OutOfSchool.WebApi/Controllers/V1/RatingController.cs#L108
Though, I will remove this option